sample_application/app_logging.py

```diff
@@ -0,0 +1,76 @@
+"""
+Utility functions for the blueprint
+"""
+
+__author__ = 'J. Elliott'
+__maintainer__ = 'J. Elliott'
+__copyright__ = 'Copyright 2015'
+__version__ = '1.0'
+__email__ = 'ads@cfa.harvard.edu'
+__status__ = 'Production'
+__credit__ = ['V. Sudilovsky']
+__license__ = 'GPLv3'
+
+from logging import StreamHandler, Formatter
+from logging.handlers import TimedRotatingFileHandler, SysLogHandler
+
+
+class Logging(object):
+    """
+    Class for Flask application logging
+
+    Supports three types of handlers:
+      File handler
+      Standard out
+      System log
+    """
+
+    def __init__(self):
```

Comment:
This looks a lot like a flask extension, so for that reason let's follow the pattern outlined in http://flask.pocoo.org/docs/0.10/extensiondev/#the-extension-code